[2020-04-06 00:17:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 00:51:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 01:49:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 01:49:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 03:14:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 03:41:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 03:59:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 04:52:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 08:04:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 08:07:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 10:14:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 12:21:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 12:21:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 14:35:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 15:43:03] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 15:43:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 15:48:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 15:48:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 15:48:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 17:15:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 19:03:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 20:10:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 20:46:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2020-04-06 22:00:22] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
